packages/dapp/components/lib/hooks/contracts/use-manager-managed.tsx

```diff
@@ -29,9 +29,10 @@ export const ManagedContractsContextProvider: React.FC<ChildrenShim> = ({ childr
   const { provider } = useWeb3();
   const deployedContracts = useDeployedContracts();
   const [managedContracts, setManagedContracts] = useState<ManagedContracts>(null);
+  const NETWORK_ANVIL_ID = 31337;
 
   useEffect(() => {
-    if (deployedContracts && provider) {
+    if (deployedContracts && provider && provider.network.chainId === NETWORK_ANVIL_ID) {
```

Comment:
added to ignore call reverts

Reply:
![call](https://github.com/ubiquity/ubiquity-dollar/assets/41552663/1afc82ab-c04d-4728-80c4-9063b6975070)